hw/xbox/chihiro.c

```diff
@@ -388,13 +388,13 @@ static void machine_set_mediaboard_filesystem(Object *obj, const char *value,
 
 static inline void chihiro_machine_initfn(Object *obj)
 {
-    object_property_add_str(obj, "mediaboard_rom",
+    object_property_add_str(obj, "mediaboard-rom",
                             machine_get_mediaboard_rom,
                             machine_set_mediaboard_rom, NULL);
     object_property_set_description(obj, "mediaboard_rom",
```

Comment:
What about these? I missed them in my first pass. Please grep / "Find" all of these strings please, to make absolutely sure you got them all.

It might be worth to even check other components in /hw/xbox by grepping for `object_property_`: `grep -n "object_property_" -R | grep _`

Reply:
Thanks!
I don't find any more occurrences (by us)